src/server/middlewares/states.ts

```diff
@@ -0,0 +1,37 @@
+import { Middleware } from 'koa'
+import { getBlogPosts } from '../utils/storyblok'
+import { getCachedTeamtailorUsers } from '../utils/teamtailor'
+
+export const addTeamtailorUsersToState: Middleware = async (ctx, next) => {
```

Comment:
I like this! If I understand correctly it means that we only load the TeamTailor users when accessing a page that requires them, right? 🙂